pages/linux/do-release-upgrade.md

```diff
@@ -1,6 +1,7 @@
 # do-release-upgrade
 
 > The Ubuntu release upgrader.
+> More information: <https://ubuntu.com/server/docs/upgrade-introduction>.
```

Comment:
Not sure about this one. It's well explained, but only one of the two flags are covered.